### ISSUE TITLE
Can't connect the room in iOS NativeDemo app

### ISSUE THREAD

**Issue by qiweiye** [EricssonResearch/openwebrtc-examples#221]
Running the NativeDemo on my iPad, input the session id. then alert dialog is shown:
 Error Domain -" NSURLErrorDomainCode= -1202 "The certificate for this server is invalid"....

**Comment by qiweiye** [comment]
![111](https://cloud.githubusercontent.com/assets/763675/25295531/b300a68a-2698-11e7-8b10-ff418d49541a.jpg)

**Comment by hammy-dev-world** [comment]
Hi,

Were you able to find the solution?

**Comment by ZhuQinglei** [comment]
Yes, I am having the same problem @qiweiye @hammy-dev-world @nevyn @alessandrod @evermeer  could you please take a look for this issue?

**Comment by MegaManX32** [comment]
@ZhuQinglei  @hammy-dev-world Same here, any solutions yet?
